Patch from Debian: "conditionally add source files", see
https://sources.debian.org/patches/k2pdfopt/2.55%2Bds-2/0012-willuslib-CMakeLists.txt-conditionally-add-source-fi.patch/

Add ../mupdf_mod/font-win32.c to get the mupdf part to compile.

Index: willuslib/CMakeLists.txt
--- willuslib/CMakeLists.txt.orig
+++ willuslib/CMakeLists.txt
@@ -9,14 +9,27 @@ include_directories(..)
 set(WILLUSLIB_SRC
     ansi.c array.c bmp.c bmpdjvu.c bmpmupdf.c dtcompress.c filelist.c
     fontdata.c fontrender.c gslpolyfit.c linux.c math.c mem.c ocr.c
-    ocrgocr.c ocrtess.c ocrwords.c pdffonts.c pdfwrite.c point2d.c
-    render.c strbuf.c string.c token.c wfile.c wgs.c wgui.c
+    ocrwords.c pdffonts.c pdfwrite.c point2d.c
+    render.c strbuf.c string.c token.c wfile.c wgui.c
     willusversion.c win.c winbmp.c wincomdlg.c wininet.c winmbox.c
     winshell.c winshellwapi.c wleptonica.c wmupdf.c wmupdfinfo.c wpdf.c
     wpdfoutline.c wpdfutil.c wsys.c wzfile.c
+    ../mupdf_mod/font-win32.c
 )
 # ocr.c  
 # win.c  winbmp.c winmbox.c wincomdlg.c wgui.c winshell.c
+
+if(HAVE_GHOSTSCRIPT)
+  set(WILLUSLIB_SRC ${WILLUSLIB_SRC} wgs.c)
+endif(HAVE_GHOSTSCRIPT)
+
+if(GOCR_FOUND)
+  set(WILLUSLIB_SRC ${WILLUSLIB_SRC} ocrgocr.c)
+endif(GOCR_FOUND)
+
+if(TESSERACT_FOUND)
+  set(WILLUSLIB_SRC ${WILLUSLIB_SRC} ocrtess.c)
+endif(TESSERACT_FOUND)
 
 if(DJVU_FOUND)
   set(WILLUSLIB_SRC ${WILLUSLIB_SRC} bmpdjvu.c)
